digitalpy/core/telemetry/impl/opentel_counter.py

```diff
@@ -1,5 +1,5 @@
 from typing import Dict
-from digitalpy.telemetry.counter import Counter
+from digitalpy.core.telemetry.counter import Counter
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I003 isort expected 1 blank line in imports, found 0